net-mail/mboxgrep/mboxgrep-0.7.9-r3.ebuild

```diff
@@ -34,11 +32,6 @@ src_prepare() {
 	eautoreconf
 }
 
-src_configure() {
-	econf \
-		$(use_with dmalloc dmalloc $(usex dmalloc yes no))
```

Comment:
Why not --without-dmalloc here to be safe? Or --with-dmalloc=no?

Reply:
same, explicitly disabled now